libs/vkd3d/command.c

```diff
@@ -5455,15 +5470,21 @@ static void STDMETHODCALLTYPE d3d12_command_list_IASetIndexBuffer(d3d12_command_
             index_type = VK_INDEX_TYPE_UINT32;
             break;
         default:
-            WARN("Invalid index format %#x.\n", view->Format);
-            return;
+            FIXME_ONCE("Invalid index format %#x. This will map to R16_UINT to match observed driver behavior.\n", view->Format);
+            /* D3D12 debug layer disallows this case, but it doesn't trigger a DEVICE LOST event, so we shouldn't crash and burn.
+             * Mark the command buffer as not having a valid index buffer. */
```

Comment:
> `* Mark the command buffer as not having a valid index buffer.

That's not what we're doing here, though.

Reply:
Derp.

Reply:
Fixed.